modules/ROOT/pages/rest-api-access.adoc

```diff
@@ -12,28 +12,42 @@ include::partial$_std-hdr-sgw.adoc[]
 include::partial$block-abstract.adoc[]
 
 :xref--pfx: {sgw--xref}:
+:tls-encryption--xref: {sgw--xref}{secure-sgw-access--page}[enforcing TLS encryption]
 
 == Overview
 
-In addition to the {rest-api--xref}, Sync Gateway provides secure access to two REST API  targeted at specific user populations and functional areas, namely the:
+Sync Gateway provides secure access to its REST APIs, namely the:
 
-* {rest-api-admin--xref}
-* {rest-api-metrics--xref}
+* {rest-api--xref}
+* {rest-api-admin--xref} -- for the administration and configuration of {sgw}
+* {rest-api-metrics--xref} -- for the monitoring of {sgw} performance metrics
 
 Each REST API is accessed through a different, <<lbl-port-cfg, user-specifiable>>, TCP port.
-This makes it easy to control their physical exposure, perhaps keeping the Admin REST API secure behind your firewall.
+This makes it easy to control their physical exposure, perhaps to keep the Admin REST API secure behind your firewall.
 
-{sgw-s} 3.0 brings additional layers of security to the Metrics and Admin interfaces by requiring Couchbase Server Role-Based Access Control (RBAC) authentication for all <<lbl-secure-users,secure API users>>, and by {sgw--xref}{security--page}[enforcing TLS encryption].
+{sgw-s} 3.0 brings additional layers of security by {tls-encryption--xref} for all API traffic and by requiring Couchbase Server Role-Based Access Control (RBAC) authentication for all <<lbl-secure-users,Admin and Metrics API users>>.
 
 RBAC user authentication enables secure remote administration of {sgw} clusters.
 This is critical in cloud native deployments.
 It provides secure and fine-grained access control.
 
 
-[#lbl-secure-users]
-== Secure API Users
+== Basic Authentication
 
-In order to submit Admin or Metrics REST API requests you must create Couchbase Server users for that purpose, and to then provide a valid set of Couchbase Server credentials for the pre-configured Couchbase Server user on each API request.
+In 3.0, TLS is enabled by default.
```

Comment:
This sounds like we're confusing TLS and admin auth. 
TLS is on the same topic of security but separate from the admin auth work. You can have admin auth without TLS and TLS without admin auth.